WordPress/src/main/java/org/wordpress/android/networking/PageStore.kt

```diff
@@ -0,0 +1,49 @@
+package org.wordpress.android.networking
+
+import kotlinx.coroutines.experimental.CommonPool
+import kotlinx.coroutines.experimental.withContext
+import org.greenrobot.eventbus.Subscribe
+import org.greenrobot.eventbus.ThreadMode
+import org.wordpress.android.fluxc.store.PostStore
+import org.wordpress.android.models.pages.PageModel
+import org.wordpress.android.fluxc.Dispatcher
+import org.wordpress.android.fluxc.generated.PostActionBuilder
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.fluxc.store.PostStore.FetchPostsPayload
+import org.wordpress.android.fluxc.store.PostStore.OnPostChanged
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlin.coroutines.experimental.Continuation
+import kotlin.coroutines.experimental.suspendCoroutine
+
+@Singleton
+class PageStore @Inject constructor(private val postStore: PostStore, private val dispatcher: Dispatcher) {
```

Comment:
Thanks :). I've thought about this and it's really just a facade that tries to hide the poor design choices on the backend. Since it's really not a real "store" I kept it here. But if you think it makes more sense I could move it (along with the model class).

Reply:
If we move it to FluxC, we might set the pattern for future Stores (and it feels like it belongs there). In future we might change the implementation to not use PostStore and the client wouldn't even need to know :-). This can be done in a future PR

Reply:
OK, let's do it in a different PR.